kolibri/core/assets/src/state/actions.js

```diff
@@ -176,9 +176,10 @@ function kolibriLogin(store, sessionPayload, isFirstDeviceSignIn) {
     .then(session => {
       store.dispatch('CORE_SET_SESSION', _sessionState(session));
       const manageURL = urls['kolibri:managementplugin:management']();
+      const deviceURL = urls['kolibri:managementplugin:device_management']();
       if (isFirstDeviceSignIn) {
         // Hacky way to redirect to content import page after completing setup wizard
-        redirectBrowser(`${window.location.origin}${manageURL}#/content`);
+        redirectBrowser(`${window.location.origin}${deviceURL}#/welcome`);
```

Comment:
Login route changed to content management, added a route to add welcome modal to the page